client/landing/stepper/declarative-flow/internals/global.scss

```diff
@@ -70,6 +70,7 @@ button {
 .import-focused,
 .plugin-bundle,
 .newsletter-setup,
+.newsletter-type,
```

Comment:
This is such a weird list of step names, doesn't make sense and smells like need to refactor something.

Reply:
Agreed, but probably something to discuss and handle separately. The way this is now, I actually think a css refactor would be kind of delicate and require a fair amount of testing across stepper flows.